update-center-source.properties

```
@@ -1,12 +1,20 @@
 plugins=aemrules,ansible,apigee,authaad,authbitbucket,authgoogleoauth,authoidc,checkstyle,citymodel,cloudformation,clover,cnesreport,cobertura,codehawk,communitybsl,communityrust,crowd,dependencycheck,detekt,devon4j,findbugs,fxcop,groovy,hadolint,ibmixaemrules,jdepend,jqassistant,lits,l10nde,l10nfr,l10nel,l10nes,l10nit,l10nja,l10nko,l10nru,l10nzh,l10nzhtw,mulesoft,mutationanalysis,mybatis,openedge,pmd,quboo,shellcheck,rci,scmcvs,scmjazzrtc,scmtfvc,scmmercurial,smells,softvis3d,sonargraphintegration,webdriver,xanitizer,yaml
 scanners=scannercli,scannermsbuild,scannerant,scannermaven,scannergradle,scannerazure,scannerjenkins
 
-devVersion=9.6-SNAPSHOT
-publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5
+devVersion=9.7-SNAPSHOT
+publicVersions=8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8,7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8
 
 ltsVersion=8.9.9
 
+9.6.description=New rich and context-specific rule descriptions, token expiration, PCI DSS report, encryption for SAML authentication, faster analysis for Java PRs and Kotlin, Kubernetes security rules, JavaScript rules for CDK S3 buckets and React, new rules for AWS in Java and for Azure in .NET
+9.6.downloadUrl=https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-9.6.0.59041.zip
+9.6.downloadDeveloperUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-developer/sonarqube-developer-9.6.0.59041.zip
+9.6.downloadEnterpriseUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-enterprise/sonarqube-enterprise-9.6.0.59041.zip
+9.6.downloadDatacenterUrl=https://binaries.sonarsource.com/CommercialDistribution/sonarqube-datacenter/sonarqube-datacenter-9.6.0.59041.zip
+9.6.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%20SONAR%20AND%20fixVersion%20%3D%209.6
```

You should use the release notes, right?

```suggestion
9.6.changelogUrl=https://sonarsource.atlassian.net/secure/ReleaseNote.jspa?projectId=10139&version=12633
```

---

As per[ this discussion](https://discuss.sonarsource.com/t/release-note-urls-in-jira-are-broken/11144/4) we earlier decided to use the issues list for release from this release. However, the one you shared is a better solution if we can make it public. Currently, that page appears to be private and needs login. What do you suggest?

---

Ah, it thought it was already solved. Well, it's not a big deal, leave it as you wrote 👍